server/database/banano/known-accounts/faucet.json

```diff
@@ -245,6 +245,11 @@
         "address": "ban_36seefx46pwcpyp6a8kukybamqioam6a7jef88s8esjpubyc8urccebjqgyj",
         "alias": "nanswap Faucet"
     },
+    {
+        "address": "ban_3dybnpbt61kokfk6u8c1uuyyed4apy8349qwb5ip9m71kschfoeukeepfzr7",
```

Comment:
Hey @MajicGit thanks for the PR & your patience here; I just back home from traveling. 

I'm a little iffy on the NanSwap Rewards entry because it _might_ be better labeled as "exchange", but seeing that it's a way for people to get payouts "faucet" is fine as well.  

Merging this in 👍